src/amp-story-embed-manager.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpStoryEmbed} from './amp-story-embed';
+
+/** @const {string} */
+const SCROLL_THROTTLE_MS = 500;
+
+export class AmpStoryEmbedManager {
+  /**
+   * @param {!Window} win
+   * @constructor
+   */
+  constructor(win) {
+    /** @private {!Window} */
+    this.win_ = win;
+
+    /** @private {?AmpStoryEmbed} */
+    this.currentEmbed_ = null;
```

Comment:
Nit: we should not write code that's only used for unit testing purposes (this and the `getEmbed()` method). Do you have another way of writing your test that wouldn't need this piece of code?
`getEmbed` returns you a pretty arbitrary embed (the last one), and `currentEmbed` doesn't make sense if there are two embeds side to side in the page

Reply:
Done.